public/gbm_cptac_2021/meta_lipidome_positive_quantification.txt

```diff
@@ -0,0 +1,10 @@
+genetic_alteration_type: GENERIC_ASSAY
+show_profile_in_analysis_tab: true
+data_filename: data_lipidome_positive_quantification.txt
+datatype: LIMIT-VALUE
+stable_id: lipidome_positive
+profile_name: Lipid abundance ratio
```

Comment:
You should indicate the mode in this entry as well. Else it will have the same name as the negative profile